apps/src/code-studio/pd/teachercon1819_registration/Joining.jsx

```diff
@@ -97,6 +97,19 @@ export default class Joining extends Teachercon1819FormComponent {
         </FormGroup>
       }
 
+      {(this.isFacilitatorApplication || this.isPartnerApplication()) &&
```

Comment:
should this be `this.isFacilitatorApplication()` ?

Reply:
d'oh, yes